src/grt/include/grt/GlobalRouter.h

```diff
@@ -289,6 +289,10 @@ class GlobalRouter : public ant::GlobalRouteSource
   void initNetlist(std::vector<Net*>& nets);
   bool makeFastrouteNet(Net* net);
   bool pinPositionsChanged(Net* net, std::vector<odb::Point>& last_pos);
+  std::vector<int> findTransitionLayers();
```

Comment:
Adding a typedef for int naming it LayerId or the like would probably help readability.